application/common/config/main.php

```diff
@@ -38,6 +33,16 @@
     ];
 }
 
+$emailServiceConfig = Env::getArrayFromPrefix('EMAIL_SERVICE_');
```

Comment:
@Schparky - The `$emailServiceConfig` variable is used before this point, up on line 30, so it looks like we should pull this instantiation code back up to before that.

Reply:
Thanks. I found that bug a few days ago. It should be fixed on `develop`.